interfaces/data/IRss.ts

```diff
@@ -4,10 +4,16 @@
 
 import Parser from 'rss-parser';
 import { IRssItem } from './IRssItem';
+import { IRssPodcastValue } from './IRssPodcast';
 
 export interface IRss extends Parser.Output<IRssItem> {
   /**
    * Copyright string.
    */
   copyright?: string;
+
+  podcast?: {
```

Comment:
Like `itunes`, adding the namespace as an object, with the values for it within that `podcast` namespace.